.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "06:00"
+  open-pull-requests-limit: 10
+  target-branch: current
```

Comment:
```suggestion
  target-branch: dev
```

Reply:
Actually it should have been removed 😉 

Bad copy & pasta actions from IO

Reply:
Even better